client/src/Components/AllVideos.js

```diff
@@ -0,0 +1,15 @@
+import VideoCard from "./VideoCard";
+
+const AllVideos = ({ videos, setVideos }) => {
+  const removeVideo = (video) =>
+    setVideos(videos.filter((oneVideo) => oneVideo !== video));
+  return (
+    <div className="row">
```

Comment:
Your code looks good to me! 
But you can improve it further by using `semantic HTML` and making it more accessible. You can use an unordered list to represent a list of videos, and each video can be represented as a list item. `ul` and `li` tags instead of `div`